lib/relationDisplay.ml

```diff
@@ -647,6 +651,16 @@ let print_dag_links conf base p1 p2 rl =
     let compare = compare
   end in
   let module M = Map.Make (O) in
+  let sps =
+    match (Util.p_getenv conf.env "sp", Util.p_getenv conf.env "spouse") with
+    | Some ("off" | "0"), _ | _, Some "off" -> false
```

Comment:
I did not try to apply the off/0 convention to spouse, only to sp.
But yes, this may seem confusing, and removing spouse altogether would clarify things